### PR TITLE
Paragraph Collection

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY
- (Edit the above to reflect status)

# Summary
- Add collections paragraph type. (Ignore the .ddev folder)

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SUL23-74
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/sites/default/settings.php [hunk 1]
```diff
@@ -764,3 +764,9 @@
  *
  * @link https://docs.acquia.com/blt/
  */
+
+// Automatically generated include for settings managed by ddev.
+$ddev_settings = dirname(__FILE__) . '/settings.ddev.php';
+if (getenv('IS_DDEV_PROJECT') == 'true' && is_readable($ddev_settings)) {
+  require $ddev_settings;
+}
```
Comment (pookmish) [inline docroot/sites/default/settings.php]: > Do not include additional settings here

Would it still work to move this into `docroot/sites/default/settings/default.local.settings.php` file?

File: .gitignore [hunk 2]
```diff
@@ -162,3 +162,4 @@ drush/sites/.checksums
 
 # Deprecation detector rules
 .rules
+sulgryphon-nextjs
```
Comment (pookmish) [inline .gitignore]: This is kinda already added at the top `frontend`. Let's just move it up there.

**Review by pookmish — CHANGES_REQUESTED** [review]
Just a couple nit picky things and then I'm good with this. The UI is a little clunky, but we can review that later.

File: docroot/sites/default/settings.php (hunk 1 above)
Comment (mdyoung3) [reply docroot/sites/default/settings.php]: I'll give it try. 

Removed!

File: .gitignore (hunk 2 above)
Comment (mdyoung3) [reply .gitignore]: Done!

**Review by pookmish — APPROVED** [review]

**Pull request merged**